src/components/commentsList/CommentsList.tsx

```diff
@@ -250,27 +251,31 @@ export function CommentsList({
                       )}
 
                       <Group gap={10}>
-                        <Button
-                          variant="transparent"
-                          p={0}
-                          onClick={() => toggleIsResolved(comment)}
-                        >
-                          <Image
-                            alt={comment.resolved ? "Resolved" : "To resolve"}
-                            src={
-                              comment.resolved ? CheckCircleFull : CheckCircle
-                            }
-                            h={24}
-                            w={24}
-                          />
-                        </Button>
-                        <Button
-                          variant="transparent"
-                          p={0}
-                          onClick={() => setCommentToDelete(comment)}
-                        >
-                          <Image alt="Delete" src={Delete} h={24} w={24} />
-                        </Button>
+                        <Tooltip label={"Toggle is resolved"}>
+                          <Button
+                            variant="transparent"
+                            p={0}
+                            onClick={() => toggleIsResolved(comment)}
+                          >
+                            <Image
+                              alt={comment.resolved ? "Resolved" : "To resolve"}
+                              src={
+                                comment.resolved ? CheckCircleFull : CheckCircle
+                              }
+                              h={24}
+                              w={24}
+                            />
+                          </Button>
+                        </Tooltip>
+                        <Tooltip label={"Delete comment"}>
+                          <Button
+                            variant="transparent"
+                            p={0}
+                            onClick={() => setCommentToDelete(comment)}
```

Comment:
wait, why is `setCommentToDelete` still here? is that up to date with develop after #80 ?